subgraph.template.yaml

```diff
@@ -10,6 +10,7 @@ dataSources:
     source:
       address: '{{court}}'
       abi: AragonCourt
+      startBlock: {{startBlock}}
```

Comment:
Nice, this is just to make the initial sync faster, right?

Reply:
yea 🎉